packages/forge2d/lib/src/dynamics/joints/distance_joint.dart

```diff
@@ -240,4 +240,15 @@ class DistanceJoint extends Joint {
 
     return C.abs() < settings.linearSlop;
   }
+
+  @override
+  void render(DebugDraw debugDraw) {
+    super.render(debugDraw);
+
+    final p1 = anchorA;
+    final p2 = anchorB;
+    final color = Color3i.zero()..setFromRGBd(0.5, 0.8, 0.8);
```

Comment:
@spydon I was thinking, perhaps we could make specifying a custom `color` possible. Not going to do it in this PR. Perhaps is a functionality that could be kept in mind. Before this refactor this was also not a thing since `color` was defined internally by `Joint`.

In addition, I don't know what do we think of replacing `Color3i` with the `Color` class we are already used to from `dart:ui`. wdyt?

Reply:
Sounds good, there is something on `Color` that is missing that `Color3i` has, but I don't remember what it was, and we can just do an extension with the things missing.